kolibri/utils/cli.py

```diff
@@ -145,6 +145,7 @@ def get_cext_path(dist_path):
 get_cext_path(sys.path[-1])
 try:
     import cryptography
+    version = cryptography.__version__ # get rid of Travis complain
```

Comment:
Is this travis complaining or the linting? You can use `  # noqa` for code lines that don't lint but aren't supposed to.

Reply:
Thanks! Will do it now